content/kubermatic/master/operation/etcd/etcd-launcher/_index.en.md

```diff
@@ -0,0 +1,105 @@
++++
+title = "[Experimental] Etcd Launcher"
+weight = 0
++++
+
+Starting with version v2.15.0, KPP introduced etcd-launcher as an experimental feature. Etcd-launcher is a lightweight wrapper around the etcd binary. It's responsible for reading information from KPP API and flixibly control how the user cluster etcd ring is started.
+
+### Spec updates
+Prior to v2.15.0, user cluster etcd ring was based on a static StatefulSet with 3 pod running the etcd ring nodes.
+With etcd-launcher, the etcd StatefulSet is updated to include:
+- An init container that is responsible for copying the etcd-launcher into the main etcd pod.
+- Additional environment variables used by the etcd-launcher and etcdctl binary for simpler operations.
+- A liveness probe to improve stability.
+- The pod command is updated to run etcd-launcher binary instead of the etcd server binary.
+
+{{% notice warning %}}
+etcd-launcher is an experimental feature. It should not be enabled unless all users clusters are operating nominally. It's possible that a user cluster etcd ring could fail to enable the feature if the etcd ring was not in a stable condition during the update.
+{{% /notice %}}
+
+Since this is an experimental feature, it's disabled by default. There are two modes to enable etcd-launcher support:
+## Enable etcd-launcher for a specific user cluster
+In this mode, the feature is only enabled for a specific user cluster. This can be done by editing the object cluster and enabling the feature flag for etcd-launcher:
+
+```bash
+$ kubectl edit cluster 27k9rzjzs7
+```
+
+Edit the cluster spec to add the `etcdLauncher` feature flag, and set it to `true`:
+
+```yaml
+spec:
+  features:
+    etcdLauncher: true
+```
+
+Once the cluster object is updated. The etcd StatefulSet will start updating the etcd ring pods one by one. Replacing the old etcd run command with the etcd-launcher.
```

Comment:
Should this have been one sentence? "Once the cluster object is updated, the etcd ...." ?